### ISSUE TITLE
[PassThePopcorn] offline

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#14591]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
not relevant
```

### Description

PTP is offline

### Logged Error Messages

```text
https://passthepopcorn.me/ cloudflare Connection timed out Error code 522
```

**Comment by garfield69** [comment]
ok I'l bite. what's funny?

**Comment by ilike2burnthing** [comment]
Looks like they've been down for about a month, but:

> The message you've been waiting for, and for some transparency’s sake (and to ease tensions due to ongoing prolonged downtime):
>
>Unfortunately some of our stuff went down together with IRC and it caught us out at a very inopportune time with people being busy due to the time of year and IRL responsibilities and ongoing personal issues.
>
>Being that we’re volunteer based, it will likely be a while before we return to full strength. The short of it is that it was just awful timing, above all else. I hope by the time we’re back, the long-promised server migration will hopefully happen that has been on hold for a while now.
>
>Once the server migration happens, things will be immediately be more stable so the extended downtime won’t keep happening like it has been.
>
>We’re not going anywhere, we’ll be back to full strength in the future and once we do, I’ll ensure we hold some type of site event to apologize for the downtime. Probably the upload contest I’ve been mentioning for a while now, or maybe an extended freeleech event.
>
>On behalf of our entire team, and entire community: thank you for the ongoing support and outpouring of love you’ve sent the staff and community’s way. It is never lost on us how important the site is. We all really miss our favorite site and we hope to be back as soon as we can.

https://ptp.trackerstatus.info/

**Comment by garfield69** [comment]
If its been over a month already I don't see the point keeping it. Despite the notice it may be a long while before its back online, we can add it back if/when its back online.

**Comment by garfield69** [comment]
v0.21.595

**Comment by ilike2burnthing** [comment]
https://www.passthepopcorn.me/

> TL;DR: PTP is still alive, and there will not be any data loss. We've been preparing to rebuild our infrastructure, but a donation drive will be necessary in order to do so.

**Comment by sanjitdutta** [comment]
to confirm... PTP will be re-added if/when it returns?

**Comment by ilike2burnthing** [comment]
Yes.

**Comment by Audionut** [comment]
It's back.

**Comment by garfield69** [comment]
https://passthepopcorn.me/
![image](https://github.com/Jackett/Jackett/assets/6933240/467b8bb7-84f2-4576-8f85-c5556506793c)

**Comment by garfield69** [comment]
online

**Comment by Audionut** [comment]
Any chance of pushing a release for this?

**Comment by ilike2burnthing** [comment]
Next release will be in ~20hrs. We only push early for breaking bugs or compatibility with FlareSolverr.

If you must have it now - https://dev.azure.com/Jackett/0f0cb3d0-9b6b-4a86-876d-0c2e17c18801/_apis/build/builds/9903/artifacts?artifactName=drop&api-version=7.1&%24format=zip

**Comment by garfield69** [comment]
v0.21.887
